docs/src/userguide/language_basics.rst

```diff
@@ -356,12 +316,29 @@ and is typically what one wants).
 If you want to use these numeric Python types simply omit the
 type declaration and let them be objects.
 
+Extention Types
+---------------
+
 It is also possible to declare :ref:`extension-types` (declared with ``cdef class`` or the ``@cclass`` decorator).
-This does allow subclasses. This typing is mostly used to access
-``cdef``/``@cfunc`` methods and attributes of the extension type.
+Those will have a behavior very close to python classes (e.g. creating subclasses), but are faster. Typing variable
```

Comment:
```suggestion
Those will have a behaviour very close to python classes (e.g. creating subclasses), but are faster. Typing variable
```

Not very important but I think we've used British/Australian/New Zealand spelling elsewhere